python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -87,9 +102,21 @@ class AssembleCellCoaddConnections(
         doc="Output multiple cell coadd",
         name="{inputWarpName}Coadd{outputCoaddSuffix}",
         storageClass="MultipleCellCoadd",
-        dimensions=("tract", "patch", "band", "skymap"),
+        dimensions=("tract", "patch", "band", "skymap", "instrument"),
```

Comment:
I don't think we can add `instrument` just yet unless we clean up the existing datasets, but this is something we want to add soon(-ish).

Reply:
@timj had mentioned about not having that info in the metadata and having to rely only the collection name. My understanding was not having this for `deepCoadd` was a mistake but something that we can fix for the cell-coadds.

https://rubin-obs.slack.com/archives/C08CANY4B6H/p1740701138145739?thread_ts=1740699057.980539&cid=C08CANY4B6H

Reply:
If we do want to add `instrument` now, that still leaves us with a question with one of the three possible answers:

1. Change the dataset type name
2. Delete all the `deepCoaddCell` datasets that Miranda and I have made, deregister and re-register the dataset type.
3. Do some database surgery to change the dimensions without affecting anything else (this is beyond me, and I don't know how easy it is)

Reply:
Tim may well win this debate in the end, but let's not do it on this PR since that does create dataset type name clashes.

Instead, let's try to get this right before we put the cell-based coadds in a DRP-v2 pipeline, since we'd want to change the dataset type name there anyway.

Reply:
Sounds good to me.